### ISSUE TITLE
/oa region temp not working from console

### ISSUE THREAD

**Issue by JerryTheSlime16** [Mindgamesnl/OpenAudioMc#320]
**Describe the bug**
When the /oa region temp <region> <src> <time> is used from console it does not play any audio to the client. It does work when executed by a player.

**Desktop (please complete the following information):**
 - Plugin Version: 6.8.3
 - Server Version: Paper version git-Paper-790 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
 - Web client used: [client.openaudiomc.net (default)]

**Comment by Mindgamesnl** [comment]
Do you have a multi-world plugin installed? If so, please check the default world you have configured in the config.

**Comment by JerryTheSlime16** [comment]
Yes; the region that I am wanting to play audio in is not in the default world.
